### PR TITLE
Fixed one broken link issue #9957

### PR DESCRIPTION
**Pull request by neerajmanivarnan**
**Notes for Reviewers**

This PR fixes #

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit d4b15d6c50590a22fd1d8c9f710dd7bf11f99771 at: https://65a37c241686f6473a008241--meshery-docs-preview.netlify.app

**Comment by neerajmanivarnan** [comment]
@leecalcote This PR fixes #9957 .Could you please review my PR.

**Comment by vishalvivekm** [comment]
> @leecalcote This PR fixes #9957 .Could you please review my PR.

would you take a look and fix other broken links on the page, as well?
Multiple links breaking at the same time, this needs investigated.

**Comment by neerajmanivarnan** [comment]
> > @leecalcote This PR fixes #9957 .Could you please review my PR.
> 
> would you take a look and fix other broken links on the page, as well?
> Multiple links breaking at the same time, this needs investigated.

Sure. There are some missing files as well

**Comment by saurabh100ni** [comment]
Hi @neerajmanivarnan   ,
Thanks for your PR! I encourage you to share this in the Website Meeting tomorrow at 6:30 PM IST. Share your screen, and walk us through it. It's cool to take and share feedback. Check the [Doc](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit) here to include this PR.

You're welcome at the meeting

Check to [community calendar](https://layer5.io/community/calendar) for more details of meetings at Layers.

**Comment by l5io** [comment]
🚀 Preview for commit f375ab84a671334f4186d2a0d2f9ce7263573718 at: https://65a413ecf7a479986dad9d26--meshery-docs-preview.netlify.app

File: docs/pages/guides/configuration-management/configuration-management.md [hunk 1]
```diff
@@ -21,8 +21,8 @@ Meshery's core construct of configuration management is that of a design. A desi
 ## Creating a Meshery Design
 
 You have your choice of how are you would like to create an infrastructure design. You can use the design configurator inside of measure, UI or extensions, like MeshMap, or you can hand code your design, using the mesh model spec, if you like.
- 
-Use Meshery UI or mesheryctl to manage [designs](../tasks/patterns.md).
+
+Use Meshery UI or mesheryctl to manage [designs](/docs/pages/tasks/patterns.md).
```
Comment (leecalcote) [inline docs/pages/guides/configuration-management/configuration-management.md:25]:
```suggestion
Use Meshery UI or mesheryctl to manage [designs]({{ site.baseurl }}/tasks/patterns).
```

@neerajmanivarnan, please accept and use this change.

1. Don't include filename extension.
1. Don't include `/docs/pages`.
1. Do use `{{ site.baseurl }}`.

**Review by leecalcote — CHANGES_REQUESTED** [review]
Apply this feedback to all links, @neerajmanivarnan

File: docs/pages/guides/configuration-management/configuration-management.md (hunk 1 above)
Comment (neerajmanivarnan) [reply docs/pages/guides/configuration-management/configuration-management.md]: The suggested changes have been made.Can you please review it again?

**Comment by l5io** [comment]
🚀 Preview for commit f13f3bdc9f60c071fa09ad55eceb2308625a71f8 at: https://65a426d21e61a3aa65ef5780--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 39c9d90ea2cdd95b072b99c53c000d0977623005 at: https://65a55a5186964d00a1b77c76--meshery-docs-preview.netlify.app

**Review by leecalcote — CHANGES_REQUESTED** [review]
There is no page published at https://docs.meshery.io/tasks/patterns, @neerajmanivarnan.

Test your changes, please.

**Comment by MUzairS15** [comment]
Merge Conflicts @neerajmanivarnan .

**Comment by MUzairS15** [comment]
// @vishalvivekm

File: docs/pages/guides/configuration-management/configuration-management.md (hunk 1 above)
Comment (leecalcote) [reply docs/pages/guides/configuration-management/configuration-management.md]: My goodness. I missed this comment previously. Yes, we're all set here. Thanks, @neerajmanivarnan 👍

**Pull request merged**